numba/targets/numbers.py

```diff
@@ -1370,6 +1370,14 @@ def boolean_to_any(context, builder, fromty, toty, val):
     asint = builder.zext(val, Type.int())
     return context.cast(builder, asint, types.int32, toty)
 
+#-------------------------------------------------------------------------------
+# Implicit cast from void pointer to numbers
```

Comment:
s/numbers/integer/ ?

Also, should this be restricted to int types that are `ptrdiff_t` sized?